iotdb/src/test/java/org/apache/iotdb/db/integration/IoTDBDeletionTest.java

```diff
@@ -0,0 +1,205 @@
+    deamon.active();
+    EnvironmentUtils.envSetUp();
+    Class.forName(Config.JDBC_DRIVER_NAME);
+    prepareSeries();
+  }
+
+  @AfterClass
+  public static void tearDown() throws Exception {
+    deamon.stop();
+    Thread.sleep(5000);
+
+    EnvironmentUtils.cleanEnv();
+  }
+
+  @Before
+  public void prepare() throws SQLException {
+    prepareData();
+  }
+
+  @After
+  public void cleanup() throws SQLException {
+    cleanData();
+  }
+
+  @Test
+  public void test() throws SQLException {
+    Connection connection = null;
+    try {
+      connection = DriverManager
+              .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root",
+                      "root");
+      Statement statement = connection.createStatement();
+      statement.execute("DELETE FROM root.vehicle.d0.s0  WHERE time <= 300");
+      statement.execute("DELETE FROM root.vehicle.d0.s1,root.vehicle.d0.s2,root.vehicle.d0.s3"
+              + " WHERE time <= 350");
+      statement.execute("DELETE FROM root.vehicle.d0 WHERE time <= 150");
+
+      ResultSet set = statement.executeQuery("SELECT * FROM root.vehicle.d0");
+      int cnt = 0;
+      while (set.next()) {
+        cnt ++;
+      }
+      assertEquals(250, cnt);
+      set.close();
+
+      set = statement.executeQuery("SELECT s0 FROM root.vehicle.d0");
+      cnt = 0;
+      while (set.next()) {
+        cnt ++;
+      }
+      assertEquals(100, cnt);
+      set.close();
+
+      set = statement.executeQuery("SELECT s1,s2,s3 FROM root.vehicle.d0");
+      cnt = 0;
+      while (set.next()) {
+        cnt ++;
+      }
+      assertEquals(50, cnt);
+      set.close();
+
+      statement.close();
+    } finally {
+      if (connection != null) {
+        connection.close();
+      }
+    }
+  }
+
+  public static void prepareSeries() throws SQLException {
+    Connection connection = null;
+    try {
+      connection = DriverManager
+              .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root",
+                      "root");
+      Statement statement = connection.createStatement();
+      for (String sql : creationSqls) {
+        statement.execute(sql);
+      }
+      statement.close();
+    } catch (Exception e) {
+      e.printStackTrace();
+    } finally {
+      if (connection != null) {
+        connection.close();
+      }
+    }
+  }
+
+  public void prepareData() throws SQLException {
+    Connection connection = null;
+    try {
+      connection = DriverManager
+              .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root",
+                      "root");
+      Statement statement = connection.createStatement();
+      // prepare BufferWrite file
+      for (int i = 201; i <= 300; i++) {
+        statement.execute(String.format(intertTemplate, i, i, i, (double) i, "\'" + i + "\'",
+                i % 2 == 0));
+      }
+      statement.execute("merge");
+      // prepare Overflow file
+      for (int i = 1; i <= 100; i++) {
```

Comment:
not overflow
un sequence file is proper

Reply:
fixed